OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -629,6 +630,165 @@ public static DreamValue NativeProc_clamp(DreamObject instance, DreamObject usr,
             }
             return new DreamValue(0);
         }
+        
+        [DreamProc("gradient")]
+        [DreamProcParameter("A", Type = DreamValueType.DreamObject)]
+        [DreamProcParameter("index", Type = DreamValueType.Float)]
+        public static DreamValue NativeProc_gradient(DreamObject instance, DreamObject usr, DreamProcArguments arguments) {
+            if (arguments.ArgumentCount <= 1) {
+                throw new IndexOutOfRangeException("Not enough arguments");
+            }
+
+            /// We dont want keyword arguments screwing with this
```

Comment:
`///` is reserved for XML comments/documentation
```suggestion
            // We dont want keyword arguments screwing with this
```

There are also several other comments in this method that should be changed.